communication/src/main/java/org/phenotips/messaging/script/AnonymousCommunicationScriptService.java

```diff
@@ -55,17 +55,13 @@
      * @param connectionId the id of the anonymous communication linking the two patients and their owners that are
      *            involved in this connection
      * @param options the mail content options selected by the user
-     * @return {@code 0} if the mail was successfully sent, other numbers in case of errors
+     * @return {@code 0} if the mail was successfully sent, http response codes in case of errors
      */
     public int sendInitialMail(String connectionId, Map<String, Object> options)
     {
-        try {
```

Comment:
It didn't seem to be catching anything really. I added some more specific catch statements inside of the ``actionManager.sendInitialMails`` call. That way it returns some http codes on failure